src/components/interactive/GroupButton.jsx

```diff
@@ -38,6 +55,19 @@ GroupButton.propTypes = {
 
   /** Specify a fds-icon component */
   Icon: PropTypes.func,
+
+  /**
+   * A component to wrap the entire group button. The
+   * `wrapper` must render the children prop passed into it
+   * to render the GroupButton.
+   */
+  Wrapper: PropTypes.elementType,
+
+  /** For internal use: Assigns the first button styling. */
+  isFirstButton: PropTypes.bool,
+
+  /** For internal use: Assigns the last button styling. */
+  isLastButton: PropTypes.bool,
```

Comment:
Just to talk this through:

So GroupButton already by "proxy"?... is already an internal button. People "can" use it, but I don't really see that being a thing, at least at the current moment. Let's say they did have a reason to use it... maybe they would have a reason to use these props? Essentially, I just wonder the usefulness of calling out that these are "internal props" when this component is private to begin with and doesn't even show up in Storybook.

Reply:
Hmm... I see what you're saying. If this component ever were public, these would no longer be private. Since it is currently private these props don't matter to them. Will remove.